### PR TITLE
User

### PR DESCRIPTION
**Pull request by autisticgeek**

### CODE REVIEW

File: model/userModel.js [hunk 1]
```diff
@@ -2,13 +2,29 @@ const mongoose = require('mongoose')
...
+    },
+
+    accountStatus: {
+        default: "new", type: String
+    },
+    isAdmin: { type: Boolean, default: false }, emailVerified: {
+        default: false, type: Boolean
+    },
+    emailToken: {
+        ype: String
```
Comment (bluehoudini) [inline model/userModel.js]: typo: "type" line 25

Comment (bluehoudini) [reply model/userModel.js]: everything is looks good

Comment (autisticgeek) [reply model/userModel.js]: thank you Will

**Comment by autisticgeek** [comment]
outdated

**Pull request closed without merging**